### PR TITLE
work around IN suggestions bug as described in DAT-58074

### PR DESCRIPTION
**Pull request by makc-dataloop**

### CODE REVIEW

File: tests/hooks/use-suggestions.spec.ts [hunk 1]
```diff
@@ -121,8 +121,13 @@ describe('use-suggestions', () => {
...
 
     it('suggestions should have the generic operators', () => {
+        const genericOperators = ['=', '!=', 'IN', 'NOT-IN']
+
         findSuggestions('Level ')
-        expect(suggestions.value).toEqual(['=', '!=', 'IN', 'NOT-IN'])
+        expect(suggestions.value).toEqual(genericOperators)
+
+        findSuggestions('Level =')
+        expect(suggestions.value).toEqual(genericOperators)
```
Comment (fadiDL) [inline tests/hooks/use-suggestions.spec.ts:130]: why should the suggestion here be the generic operators ?

File: tests/hooks/use-suggestions.spec.ts [hunk 2]
```diff
@@ -141,13 +146,12 @@ describe('use-suggestions', () => {
...
-            expect(suggestions.value).toEqual([
-                `'high'`,
-                `'medium'`,
-                `'low'`,
-                30
-            ])
+            expect(suggestions.value).toEqual(levelValues)
+
+            findSuggestions('Level IN ')
+            expect(suggestions.value).toEqual(levelValues)
```
Comment (fadiDL) [inline tests/hooks/use-suggestions.spec.ts:154]: please also test the case where we have `Level IN 'high',`
That should also show the level values too in suggestion ( in the future we want to filter out the selected values already but thats a nice to have )

**Review by fadiDL — CHANGES_REQUESTED** [review]

File: tests/hooks/use-suggestions.spec.ts (hunk 2 above)
Comment (makc-dataloop) [reply tests/hooks/use-suggestions.spec.ts:154]: done

File: tests/hooks/use-suggestions.spec.ts (hunk 1 above)
Comment (makc-dataloop) [reply tests/hooks/use-suggestions.spec.ts:130]: @guyDataloop request (if we press backspace after applying an operator from suggestions, we should see a list of operators)

Comment (fadiDL) [reply tests/hooks/use-suggestions.spec.ts:130]: @makc-dataloop would this still add a space where needed?

**Review by fadiDL — APPROVED** [review]

File: tests/hooks/use-suggestions.spec.ts (hunk 1 above)
Comment (makc-dataloop) [reply tests/hooks/use-suggestions.spec.ts:130]: @fadiDL this should not change the behavior for `Level = ` since we have a test for that and it passes

**Pull request merged**